<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>charakterform</class>
 <widget class="QWidget" name="charakterform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Name und Beschreibung</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Geben sie einen Namen und eine Beschreibung ein. * kennzeichnet ein Pflichtfeld.</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Name*</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelBeschreibung">
        <property name="text">
         <string>Beschreibung</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="textEditBeschreibung"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="abbrechenButton">
       <property name="text">
        <string>abbrechen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="weiterButton">
       <property name="text">
        <string>weiter &gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
